### ISSUE TITLE
Ticket to update the BSP view 

### ISSUE THREAD

**Issue by prayagd** [pendulum-chain/portal#599]
As Forex AMM user, i should be able to see all the details of the BSP pool for ex. the TVL of the pool, APR of the pool 

## Acceptance criteria
* If the user navigates to Backstop pool in Forex AMM
* They should see TVL of the pool (Total LP tokens in the pool), APR of the pool and their LP token balance (already shown) on the page itself

**Comment by prayagd** [comment]
Q @TorstenStueber : Does the BSP pool have an APR? and can we show that?

@Sharqiewicz Would you need design for this? If yes @prayagd would add Low fidelity wireframe for it. here is the Nabla reference
<img width="1007" alt="Screenshot 2024-10-16 at 12 48 16 PM" src="https://github.com/user-attachments/assets/4785c04a-2183-42e1-a435-585cc806ec0f">

**Comment by prayagd** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/portal/599?planning-poker) with Zenhub @ebma @Sharqiewicz @TorstenStueber

**Comment by ebma** [comment]
I assume that the APR for the backstop pool is again data exposed by the indexer so we need to change that prior to any change in the UI.

**Comment by TorstenStueber** [comment]
@prayagd yes, it has an APR and we already calculate that in the indexer and our indexer query in the Nabla UI of the Portal 
![Uploading Screenshot 2024-10-16 at 06.22.20.png…]()
already returns it. So that should be easy to implement.

**Comment by TorstenStueber** [comment]
![Screenshot 2024-10-16 at 06 22 20](https://github.com/user-attachments/assets/251629ca-02bc-4800-86c6-a5dfe103af20)

**Comment by ebma** [comment]
Ah right, it's already available. Great. I assume it's the 7d APR though as that was also the value we have on the swap pools. Important to keep in mind for the UI.

**Comment by TorstenStueber** [comment]
Yes, it is based on 7 days.
